src/app/shared/shared.module.ts

```diff
@@ -1,18 +1,21 @@
-import { CommonModule } from '@angular/common';
-import { NgModule } from '@angular/core';
-import { FormsModule } from '@angular/forms';
-import { MatDialogModule } from '@angular/material/dialog';
-import { DefaultNullOrEmpty } from '@app/pipes/DefaultNullOrEmptyPipe';
-import { FilterRecordPipe } from '@app/pipes/FilterRecordPipe';
-import { KeysPipe } from '@app/pipes/keysPipe';
-import { OrderByStatusPipe } from '@app/pipes/OrderByStatusPipe';
-import { IsPrimaryVrmPipe } from '../pipes/IsPrimaryVrmPipe';
-import { AdrReasonModalComponent } from './adr-reason-modal/adr-reason-modal.component';
-import { ErrorSummaryComponent } from './error-summary/error-summary.component';
-import { PageNotFoundComponentComponent } from './page-not-found-component/page-not-found-component.component';
-import { PendingChangesService } from './pending-changes-service/pending-changes.service';
-import { PreventLeavePageModalComponent } from './prevent-page-leave-modal/prevent-leave-page-modal.component';
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {MatDialogModule} from '@angular/material/dialog';
+import {DefaultNullOrEmpty} from '@app/pipes/DefaultNullOrEmptyPipe';
+import {FilterRecordPipe} from '@app/pipes/FilterRecordPipe';
+import {KeysPipe} from '@app/pipes/keysPipe';
+import {OrderByStatusPipe} from '@app/pipes/OrderByStatusPipe';
+import {IsPrimaryVrmPipe} from '../pipes/IsPrimaryVrmPipe';
+import {AdrReasonModalComponent} from './adr-reason-modal/adr-reason-modal.component';
+import {ErrorSummaryComponent} from './error-summary/error-summary.component';
+import {PageNotFoundComponentComponent} from './page-not-found-component/page-not-found-component.component';
+import {PendingChangesService} from './pending-changes-service/pending-changes.service';
+import {PreventLeavePageModalComponent} from './prevent-page-leave-modal/prevent-leave-page-modal.component';
 import {SortByDatePipe} from '@app/pipes/SortByDatePipe';
+import {AccordionComponent} from '@app/shared/libraries/accordion/accordion.component';
```

Comment:
have a module file inside the library folder and let the accordion be part of that module. Otherwise the share.module will get to be a mess if we keep adding lots o reusables. We need to start going modular groups